source/Interfaces/MainInterface.cpp

```diff
@@ -17,7 +17,13 @@ namespace i_2D {
     MainInterface::MainInterface(size_t id, const std::string &name) : InterfaceBase(id, name),
                                                                        mWindow(sf::VideoMode({1000, 800}),
                                                                                "Maze Window") {
-        mMenu.initialize();
+        ///Font for all objects
+        if(!mFont.loadFromFile("../../assets/font/ArialNarrow7.ttf")){
+            std::cout << "Error loading font file" << std::endl;
```

Comment:
This isn't critical, but it may be useful to print the font filename too. That way, if that file is ever missing, the user will see that without needing to dig into this code.